pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -817,13 +817,15 @@
         "1.1.37",
         "1.2.0",
         "1.1.36",
-        "1.3.0"
+        "1.3.0",
+        "1.3.1"
       ],
-      "BaselineVersion": "1.3.0",
+      "BaselineVersion": "1.3.1",
```

Comment:
Helping is different from requiring the update.  You're effectively creating a one-way patch family dependency by doing this.  We only do that in servicing if it is **required**.